src/terminal/parser/ut_parser/InputEngineTest.cpp

```diff
@@ -1055,14 +1055,14 @@ void InputEngineTest::VerifySGRMouseData(const std::vector<std::tuple<SGR_PARAMS
     SGR_PARAMS input;
     MOUSE_EVENT_PARAMS expected;
     INPUT_RECORD inputRec;
-    for (size_t i = 0; i < testData.size(); i++)
+    for (const auto& i : testData)

```

Comment:
Can this not be `for (const auto& [sgrParam, mouseEventParam]) : testData` to split the tuple up all at once and skip the `std::get`?